spot_tools_ros/src/spot_tools_ros/spot_executor_ros.py

```diff
@@ -299,6 +300,11 @@ def tf_lookup_fn(parent, child):
             10,
         )
 
+        if not use_fake_spot_interface:
```

Comment:
I think we want to delete this block of code? We removed the `publish_pose` function itself, and I think we decided we didn't need this currently.